deployment/terraform/assets/variables.tf

```diff
@@ -62,6 +62,7 @@ variable "db_engine_version" {
   default = {
     "aurora-mysql"      = "5.7.mysql_aurora.2.09.0"
     "aurora-postgresql" = "11.7"
+    "cockroachdb"       = "20.1.2"
```

Comment:
This version is not used anywhere, is to avoid failing to fail on terraform template filling, because is needed a version for each database (because RDS). I think we can solve that, or maybe put here something like `"cockroachdb" = "*"`.